src/app/cli/src/cli_parser.rs

```diff
@@ -297,6 +301,22 @@ pub fn cli() -> Command {
                             kamu delete my.dataset.%
                         "#
                     )),
+                Command::new("e2e")
```

Comment:
I think it will be better to make this a sub-command under the `system` command:
- I think we still do completions even for hidden commands/arguments
- I would like to keep the top-level commands space very minimal and clean

Reply:
Good points -- moved, done